packages/nhost-js/src/clients/functions.ts

```diff
@@ -54,6 +55,25 @@ export class NhostFunctionsClient {
     })
   }
 
+  /** @deprecated Axios will be replaced by cross-fetch in the near future. Only the headers configuration will be kept. */
+  async call<T = unknown, D = any>(
```

Comment:
Can't we just merge this and the other deprecated function to a single one?

For example:

```ts
async call<T = unknown, D = any>(
  url: string,
  data?: D,
  config?: (RestrictedFetchConfig | AxiosConfig) & { useAxios?: true }
): Promise<DeprecatedFunctionCallResponse<T>>
```